content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -364,23 +320,45 @@ const getUser = await prisma.user.findUnique({
 })
 ```
 
-### Relation count
+## Relation count
 
 In [3.0.1](https://github.com/prisma/prisma/releases/3.0.1) and later, you can [`include` or `select` a count of relations](aggregation-grouping-summarizing#count-relations) alongside fields - for example, a user's post count.
 
-### Filter a list of relations
```

Comment:
Why are these not under "nested reads" any more?
They are not on the same level as "Nested reads" and "Nested writes".

Reply:
Extra note: The output for "Relation Count" could also be improved as it's inaccurate. It's missing the other user fields such as name and email.